ACKategoriesCore/ACKUrl.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ACKUrl.swift
+//  ACKategories
+//
+//  Created by Alexandr Grigoryev on 21.02.2023.
+//
+
+import Foundation
+
+public struct ACKUrl: RawRepresentable {
```

Comment:
Also this struct should be `Codable`, as it is already `RawRepresentable` it should take no extra implementation. Also it would be good to add a test for encoding/decoding.